packages/easy_logging/easy_logging.0.5/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Module to log messages. Aimed at being both powerful and easy to use"
+
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.0"}
```

Comment:
```suggestion
  "dune" {build & >= "1.6.3"}
```
According to the `dune-project` file, this package and the other one require dune >= 1.6.3

Reply:
Thank you @kit-ty-kate.